Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/IUnityProjectExporter.cs

```diff
@@ -67,6 +60,8 @@ public interface IUnityProjectExporter
         /// <param name="path">The <see cref="FileInfo"/> representing where this props file will be written.</param>
         /// <param name="scriptingBackend">The scripting backend for the platform props.</param>
         IWSAPlayerPlatformPropsExporter CreateWSAPlayerPlatformPropsExporter(FileInfo path, ScriptingBackend scriptingBackend);
+
+        ICSharpProjectExporter CreateCSharpProjectExporter(FileInfo filePath, DirectoryInfo generatedProjectFolder, bool isGenerated);
```

Comment:
Add a summary, to match the other methods in this interface?

Reply:
Ah yeah, good call!